### PR TITLE
Release notes for April 29 release

### PR DESCRIPTION
**Pull request by taeold**
This release includes a patch update to the firestore-bigquery-export extension with some bug fixes.

### CODE REVIEW

File: firestore-bigquery-export/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Version 0.1.14
+
+fixed - Updated file naming for bq imports (#589)
+
+fixed - Remove double quotes on string arrays (#614)
```
Comment (jhuleatt) [inline firestore-bigquery-export/CHANGELOG.md:5]: these are both updates to npm scripts that don't affect the extension itself, so it doesn't actually need a version bump or re-release. Just needs an npm script release (see "Publish an npm package" in release playbook)

**Comment by jhuleatt** [comment]
@taeold apologies for not noticing this sooner, but if the only PRs in this release are for the BQ import and gen-schema-views scripts, then we don't need to bump the versions of any extensions. We should still sync next to master to get the histories realigned and get the latest code for the scripts into master

File: firestore-bigquery-export/CHANGELOG.md (hunk 1 above)
Comment (taeold) [reply firestore-bigquery-export/CHANGELOG.md:5]: Got it! Thanks for the explanation.

**Comment by taeold** [comment]
Closing per @jhuleatt's instructions. Will manually publish the import and gen schema scripts.

**Pull request closed without merging**